src/test/system_tests/sanity_build_test.js

```diff
@@ -4,114 +4,96 @@
 const _ = require('lodash');
 const api = require('../../api');
 const dbg = require('../../util/debug_module')(__filename);
-const ops = require('../utils/basic_server_ops');
 const argv = require('minimist')(process.argv);
-const blobops = require('../utils/blobops');
+
+if (process.env.SUPPRESS_LOGS) {
+    dbg.set_level(-5, 'core');
+}
 
 const { CloudFunction } = require('../utils/cloud_functions');
 const { BucketFunctions } = require('../utils/bucket_functions');
 
 const TEST_CTX = {
     client: null,
-    bucketfunc: null,
-    cloudfunc: null,
     aws_connection: null,
     azure_connection: null,
     compatible_v2: null,
     compatible_v4: null,
+    s3_endpoint: '',
```

Comment:
should we divide it into two? s3_ip and s3_port (on line 46 it's been built with those two args).

Reply:
what is the value of keeping them separated and then concat each time we need the endpoint?

Reply:
true, they should be together eventually. the question is where.
it can go both ways...